### PR TITLE
tick tock goes the clock

### PR DESCRIPTION
**Pull request by MadmanMartian**
Adds a grandfather clock. Sprite sourced from @jknpj which was from bay

### CODE REVIEW

**Comment by jknpj** [comment]
Sprite sourced from bay

File: code/datums/supplypacks.dm [hunk 1]
```diff
@@ -179,6 +179,14 @@ var/list/all_supply_groups = list("Supplies","Clothing","Security","Hospitality"
 	containername = "showers crate"
 	group = "Supplies"
 
+/datum/supply_packs/clock
+	name = "Grandfather Clock"
+	contains = list(/obj/structure/clock/unanchored)
+	cost = 40
+	containertype = /obj/structure/largecrate
+	containername = "Ticking Crate"
```
Comment (jknpj) [inline code/datums/supplypacks.dm]: "ticking crate"

**Comment by IzayaChan** [comment]
does it tick and tock

**Comment by jknpj** [comment]
@MadmanMartian was seeking for a proper sfx for that

**Pull request merged**